src/include/sof/common.h

```diff
@@ -15,7 +15,9 @@
 #define __must_check __attribute__((warn_unused_result))
 
 /* Align the number to the nearest alignment value */
+#ifndef IS_ALIGNED
```

Comment:
Much better to test for __ZEPHYR__ instead of the macro itself.  Otherwise you create heisenbugs where "which version" of the macro gets expanded depends in essentially arbitrary ways  based on the order of header inclusions in your particular file.  In this case it's benign, as the macro is simple and the implementations are surely compatible.  But in general this kind of thing is bad practice.  The preprocessor is dangerous, treat it with care.